GitDepend/Commands/LogCommand.cs

```diff
@@ -1,34 +1,54 @@
-﻿using GitDepend.CommandLine;
+﻿using System;
+using GitDepend.CommandLine;
+using GitDepend.Resources;
 using GitDepend.Visitors;
 
 namespace GitDepend.Commands
 {
     /// <summary>
     /// This command will do a git log on the named dependencies.
     /// </summary>
-    public class LogCommand : NamedDependenciesCommand<LogSubOptions>
+    public class LogCommand : ICommand
```

Comment:
Why are you removing the `NamedDependenciesCommand`? There is some built-in functionality that I think this command really needs to have.